### ISSUE TITLE
Edizon updater broken?

### ISSUE THREAD

**Issue by jhicken** [WerWolv/EdiZon#86]
I am using the latest nightly.

It looks like the update server for Edizon is having issues... In the app, it shows a 301 html response. Which is permanently moved. Perhaps Edizon does not know how to follow a redirect? I didnt do a MIM to see what request it was. That said I assume you will know what url is being requested when you go to the about page in the app. but its 💥.

**Comment by WerWolv** [comment]
Yeah the API service is down right now and I didn't have the time really to fix it. I'll do my best to get it back up as soon as possible 👍

**Comment by jhicken** [comment]
Cool thanks for the update.
